README.md

```diff
@@ -202,6 +204,17 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `CARPET-ADDONS-NOT-FOUND`
 
+### portalCreativeDelay
+
+Amount of delay ticks to use a nether portal in creative
+
+- Type: `Integer`
+- Default value: `1`
+- Suggested options: `1`, `40`, `80`, `72000`
+- Categories: `CREATIVE`, `CARPET-ADDONS-NOT-FOUND`
```

Comment:
Categories should be `FEATURE`, `EXPERIMENTAL`, `CARPET-ADDONS-NOTFOUND` to match the spectator portal rule.

Reply:
Maybe we even need a new `SPECTATOR` category? Assuming this isn’t in base carpet of course. If so, then the `FEATURE` and `EXPERIMENTAL` categories here would not be needed and just have `SPECTATOR` instead.  We’d also have to change `FEATURE` to `SPECTATOR` in the `spectatorPlayersUsePortals` rule.